scss/tech.scss

```diff
@@ -1,39 +1,34 @@
 
 .techs-container {
-  display: flex;
-  flex-wrap: wrap;
-  margin-left: 68px;
+  display: grid;
+  grid-template-areas: 
+    "main-tech tech tech"
+    "main-tech tech tech";
   width: $content-size;
 }
 
 .tech {
   align-items: center;
-  background-color: $green;
-  border-radius: 4px;
   display: flex;
   flex-direction: column;
-  margin: 20px;
-  padding: 10px 30px;
+  padding: 30px 30px;
   position: relative;
   text-align: center;
   width: $card-size;
   
-  &::before {
+  &:first-child {
+    align-self: center;
+    grid-area: main-tech;
```

Comment:
To avoid using `grid-area`, use `grid-row: 1 / 3;` so the tech will occupy the first column :smile_cat: